### DISCUSSION TITLE
OPTI Wiki Problem

### DISCUSSION THREAD

**Discussion by alfahri19** [coin-or/Ipopt#551] [Q&A] [answered]
Dear all,

Currently doing research on nonlinear problems and trying to solve them using OPTI Toolbox. For several weeks I tried to access the website of OPTI Wiki but I found an error. I just need to read the material provided there. Is there anybody who could help with this problem? Thank you in advance.

**Comment by svigerske** [comment, marked as answer]
Noone here, I'm afraid. As far as I know, the OPTI Toolbox is, sadly, no longer maintained. So could be that the website was eventually shut down.
